### PR TITLE
Release 0.2

### PR DESCRIPTION
**Pull request by neutrinog**
Fixes "all the things" for Resource Containers.

> This pull request is open to allow ongoing reviews. Please don't merge just yet.

### CODE REVIEW

File: docs/container_types.rst [hunk 1]
```diff
@@ -235,4 +229,46 @@ The config.yml indicates recommended and dependent modules:
         dependencies: 
           - "figs-sentences"
 
-Dependencies are slugs's of modules that should be read before this one. Recommendations are modules that would likely benefit the reader next.
+Dependencies are :ref:`slug` s of modules that should be read before this one. Recommendations are modules that would likely benefit the reader next.
+
+.. _types-usfm:
+
+USFM (usfm)
```
Comment (jag3773) [inline docs/container_types.rst]: Let's add a link to the USFM standard, http://ubsicap.github.io/usfm/

File: docs/linking.rst [hunk 2]
```diff
@@ -1,187 +1,186 @@
...
 Linking
 =======
 
-A resource container link provides navigable directions to a Resource Container.
+A Resource Container (RC) link allows one RC to reference content from another RC.
+
+All RC links follow a very simple structure in two different flavors.
 
-All resource container links follow a very simple structure in two different flavors: Anonymous and Titled. The link path is the Resource Container slug where the slug delimiter is replaced by a slash ``/`` character.
+* **Anonymous links** - have no title and are declared by enclsing the link in double brackets
```
Comment (jag3773) [inline docs/linking.rst]: Typo on 'enclsing'.

File: docs/linking.rst [hunk 3]
```diff
@@ -1,187 +1,186 @@
...
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Book example:
+
+- ``[Genesis 1:2](01/02)``
+
+Links to any language
+~~~~~~~~~~~~~~~~~~~~~
 
-Some times you may not wish to restrict the linked resource to a particular language. In that case you may exclude the language code from the beginning of the path and place an extra slash ``/`` in it's place.
+At times you may not wish to restrict the link to a partuclarl language of the RC.
```
Comment (jag3773) [inline docs/linking.rst]: Typo on 'partuclarl'

File: docs/linking.rst [hunk 4]
```diff
@@ -1,187 +1,186 @@
...
 - ``[[//ta-vol1/translate/man/translate-unknowns]]``
 - ``[Translate Unknowns](//ta-vol1/translate/man/translate-unknowns)``
 
 .. _linking-bible-refs:
+
 Automatically Linking Bible References
 --------------------------------------
 
-Bible references in any resource container should be automatically converted into resolvable links according to the linking rules for **book** resource types. Of course, if the reference is already a link nothing needs to be done.
+Bible references in any RC should be automatically converted into resolvable links according to the linking rules for **book** resource types.
```
Comment (jag3773) [inline docs/linking.rst]: Typo on resolvable

File: docs/manifest.rst [hunk 5]
```diff
@@ -1,122 +1,86 @@
...
+            direction: 'ltr'
+        source:
+        -
+            language: 'en'
+            identifier: 'en-asv'
+            version: '1901'
+        rights: 'CC BY-SA 4.0'
+        creator: 'Wycliffe Associates'
+        contributor:
+            - 'NamesOfContributors'
```
Comment (jag3773) [inline docs/manifest.rst]: Let's put two entries in here just so the example is clear that it's a list and not a string with multiple names.

File: docs/manifest.rst [hunk 6]
```diff
@@ -1,122 +1,86 @@
...
+
+Resource Containers (RCs) have a manifest.yml that describes it's content and structure.
+Most of the information adheres to the `Dublin Core Meta Data Initiative <http://dublincore.org/documents/dcmi-terms>`_
+and can be found nested within the ``dublin_core`` key.
+
+.. code-block:: yaml
+
+    ---
+
+    dublin_core:
```
Comment (jag3773) [inline docs/manifest.rst]: Can we sort these A-Z please?

File: docs/linking.rst (hunk 4 above)
Comment (neutrinog) [inline docs/linking.rst]: resolvable is the correct spelling. Do you have a suggestion?

File: docs/manifest.rst (hunk 6 above)
Comment (neutrinog) [inline docs/manifest.rst]: done

File: docs/container_types.rst (hunk 1 above)
Comment (neutrinog) [inline docs/container_types.rst]: done

File: docs/linking.rst (hunk 3 above)
Comment (neutrinog) [inline docs/linking.rst]: done

File: docs/linking.rst (hunk 2 above)
Comment (neutrinog) [inline docs/linking.rst]: done.

File: docs/manifest.rst (hunk 5 above)
Comment (neutrinog) [inline docs/manifest.rst]: done.

**Comment by jag3773** [comment]
👍

**Pull request merged**